vault/resource_pki_secret_backend_config_est_test.go

```diff
@@ -0,0 +1,203 @@
+					resource.TestCheckResourceAttr(dataName, consts.FieldAuthenticators+".#", "1"),
+					resource.TestCheckResourceAttr(dataName, consts.FieldAuthenticators+".0.%", "2"),
+					resource.TestCheckResourceAttr(dataName, consts.FieldAuthenticators+".0.cert.%", "2"),
+					resource.TestCheckResourceAttr(dataName, consts.FieldAuthenticators+".0.cert.accessor", "test"),
+					// @TODO add these back in when Vault 1.16.3 is released (https://github.com/hashicorp/vault-enterprise/pull/5785)
+					// resource.TestCheckResourceAttr(dataName, consts.FieldAuthenticators+".0.cert.cert_role", "a-role"),
+					resource.TestCheckResourceAttr(dataName, consts.FieldAuthenticators+".0.userpass.%", "1"),
+					resource.TestCheckResourceAttr(dataName, consts.FieldAuthenticators+".0.userpass.accessor", "test2"),
+					resource.TestCheckResourceAttr(dataName, consts.FieldEnableSentinelParsing, "true"),
+					resource.TestCheckResourceAttr(dataName, consts.FieldAuditFields+".#", "20"),
+					resource.TestCheckResourceAttrSet(dataName, consts.FieldLastUpdated),
+				),
+			},
+			testutil.GetImportTestStep(resourceBackend, false, nil),
+		},
+	})
+}
+
+func testAccPKISecretBackendConfigEstComplete(pkiPath string) string {
+	return fmt.Sprintf(`
+resource "vault_mount" "test" {
+	path        = "%s"
+	type        = "pki"
+    description = "PKI secret engine mount"
+}
+
+resource "vault_pki_secret_backend_role" "est_role" {
+  backend = vault_mount.test.path
+  name = "est-role"
+  ttl = 3600
+  key_type = "ec"
+  key_bits = "256"
+}
+
+resource "vault_pki_secret_backend_role" "est_role_2" {
+  backend = vault_mount.test.path
+  name = "est-role-2"
+  ttl = 3600
+  key_type = "ec"
+  key_bits = "256"
+}
+
+resource "vault_pki_secret_backend_config_est" "test" {
+  backend = vault_mount.test.path
+  enabled = true
+  default_mount = true
+  default_path_policy = format("role:%%s", vault_pki_secret_backend_role.est_role.name)
+  label_to_path_policy = {
+     "test-label": "sign-verbatim",
+     "test-label-2": format("role:%%s", vault_pki_secret_backend_role.est_role_2.name)
+  }
+  authenticators { 
+	# @TODO add these back in when Vault 1.16.3 is released (https://github.com/hashicorp/vault-enterprise/pull/5785)
+	# cert = { "accessor" = "test", "cert_role" = "a-role" }
+	cert = { "accessor" = "test", "cert_role" = "" }
+	userpass = { "accessor" = "test2" } 
+  }	
+  enable_sentinel_parsing = true
+  audit_fields = ["csr", "common_name", "alt_names", "ip_sans", "uri_sans", "other_sans",
+                  "signature_bits", "exclude_cn_from_sans", "ou", "organization", "country", 
+                  "locality", "province", "street_address", "postal_code", "serial_number",
+                  "use_pss", "key_type", "key_bits", "add_basic_constraints"]
+}
+
+data "vault_pki_secret_backend_config_est" "test" {
+  depends_on = [vault_pki_secret_backend_config_est.test]
+  backend = vault_mount.test.path
```

Comment:
nit: since the `backend` field for the config resource is also pointing to the `vault_mount` path, we can have a cleaner dependency graph and remove `depends_on` from the line above:
```suggestion
  backend = vault_pki_secret_backend_config_est.test.backend
```

Reply:
Fixed in https://github.com/hashicorp/terraform-provider-vault/pull/2246/commits/5f5961a4a31806fc4509223e1234830c7a669590